#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_windowmenus.dpatch by  <gaudenz@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use internal about dialog, not gtk's

@DPATCH@
diff -urNad passepartout-0.6-3~/src/pptout/windowmenus.cc passepartout-0.6-3/src/pptout/windowmenus.cc
--- passepartout-0.6-3~/src/pptout/windowmenus.cc	2005-12-21 02:29:14.000000000 +0100
+++ src/pptout/windowmenus.cc	2005-12-21 02:29:20.000000000 +0100
@@ -209,8 +209,8 @@
   list_help.push_back(MenuElem("_User's guide", AccelKey("F1"),
 			       sigc::ptr_fun(&open_docs)));
   list_help.push_back(MenuElem("_About", 
-			       mem_fun(AboutDialog::instance(), 
-                                       &AboutDialog::show_all)));
+			       mem_fun(::AboutDialog::instance(), 
+                                       &::AboutDialog::show_all)));
   list_help.push_back(SeparatorElem());
   list_help.push_back(MenuElem("_Homepage", 
 			       sigc::ptr_fun(&open_homepage)));
